src/components/AccountGraphCard.js

```diff
@@ -87,7 +87,7 @@ class AccountGraphCard extends PureComponent<Props, State> {
           isInteractive={isAvailable}
           isLoading={!isAvailable}
           height={100}
-          width={Dimensions.get("window").width - 40}
+          width={Dimensions.get("window").width - 32}
```

Comment:
Shouldn't this use `getWindowDimenions`? 
There's some platform specific stuff for android 🤷‍♂️